tools/docker-network/tests/dockerframework.go

```diff
@@ -284,23 +297,36 @@ func (d *DockerTestFramework) AccountsFromNodes(nodes ...*Node) []string {
 	return accounts
 }
 
-func (d *DockerTestFramework) SetIssueCandidacyPayload(issueCandidacyPayloadMap map[string]bool) {
-	cmd := exec.Command("docker", "compose", "up", "-d")
+func (d *DockerTestFramework) StartIssueCandidacyPayload(nodeNames ...string) {
```

Comment:
I think it's better design to pass a slice/variadic of nodes instead of the string/alias. That way you can also pass in a variable of a node and don't need to handle the string. When calling it with the aliases you just need to wrap it in `d.Nodes("V1", "V2", "V3")`.